### DISCUSSION TITLE
Mouse jumps away sometimes

### DISCUSSION THREAD

**Discussion by slockers37** [asmvik/yabai#1711] [Q&A] [answered]
Hello, I am very new to this tiling WM thing so I might be very inexperienced about things. 

I am using Arc Browser and when the move the mouse cursor to the PiP(Picture in Picture) window and try and click something, the mouse moves away to another corner of the app, is there a fix for this? This happens with extension windows as well.

**Comment by asmvik** [comment, marked as answer]
Sounds like your browser is doing something wonky. You can solve this by disabling mouse follows focus. `yabai -m config mouse_follows_focus off`

Optionally, you can disable `mouse_follows_focus` for Arc Browser only, using window rules. Something like: `yabai -m rule --add app="^Arc$" mouse_follows_focus=off`

Not sure if the name is Arc or Arc Browser. Use the query system to figure this out: `yabai -m query --windows`
